recipe/cakephp.php

```diff
@@ -25,15 +25,15 @@
  * Create plugins' symlinks
  */
 task('deploy:init', function () {
-    run('{{release_or_current_path}}/bin/cake plugin assets symlink');
+    run('{{bin/php}} {{release_or_current_path}}/bin/cake.php plugin assets symlink');
```

Comment:
.php?

Reply:
yes, this file https://github.com/cakephp/app/blob/4.x/bin/cake.php called directly instead of the wrapper shell provided. It would allow you to use the right version of PHP configured in {{bin/php}} instead of relying on the shell to locate the default configured php version in the target env.